### PR TITLE
feat: [FFM-4854]: Add get-started example

### PR DESCRIPTION
**Pull request by knagurski**
This PR adds a Get Started example project and updates the README with the correct docs links.

### CODE REVIEW

File: examples/get-started/src/vite-env.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```
Comment (crystal-fong) [inline examples/get-started/src/vite-env.d.ts:1]: is this needed if it commented out?

File: examples/get-started/vite.config.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+// https://vitejs.dev/config/
```
Comment (crystal-fong) [inline examples/get-started/vite.config.ts:4]: is this here for a quick access to reference?

File: examples/get-started/src/components/AllFlagsUsingHOC.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { FC } from 'react'
+import { withFeatureFlags } from '@harnessio/ff-react-client-sdk'
+import Loader from './Loader/Loader'
+
+const AllFlagsUsingHOC: FC = withFeatureFlags(({ flags, loading }) => (
+  <section>
+    <h2>All feature flags</h2>
```
Comment (crystal-fong) [inline examples/get-started/src/components/AllFlagsUsingHOC.tsx:7]: nitpick:
```suggestion
    <h2>All Feature Flags</h2>
```

File: examples/get-started/src/components/AllFlagsUsingHook.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import { useFeatureFlags } from '@harnessio/ff-react-client-sdk'
+import Loader from './Loader/Loader'
+
+const AllFlagsUsingHook: FC = () => {
+  const allFlags = useFeatureFlags()
+  const loading = !!Object.keys(allFlags).length
+
+  return (
+    <section>
+      <h2>All feature flags</h2>
```
Comment (crystal-fong) [inline examples/get-started/src/components/AllFlagsUsingHook.tsx:11]: same again, nitpick
```suggestion
      <h2>All Feature Flags</h2>
```

File: examples/get-started/src/App.tsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    <FFContextProvider
+      apiKey="YOUR_API_KEY"
+      target={target}
+      fallback={<Loader />}
+      async={useAsyncMode}
+    >
+      <article>
+        <h1>Feature Flags</h1>
+        <button onClick={handleAsyncModeToggle}>
+          {useAsyncMode ? 'Use synchronous mode' : 'Use asynchronous mode'}
```
Comment (crystal-fong) [inline examples/get-started/src/App.tsx:42]: should this be the other way around?

File: examples/get-started/src/App.module.css [hunk 6]
```diff
@@ -0,0 +1,46 @@
+article {
+  display: grid;
+  gap: 2rem;
+  grid-template:
+    'Header Toggle'
+    'Sections Sections';
```
Comment (crystal-fong) [inline examples/get-started/src/App.module.css:6]: interesting name lol

**Review by Kira-Jennings — APPROVED** [review]

File: examples/get-started/src/vite-env.d.ts (hunk 1 above)
Comment (knagurski) [reply examples/get-started/src/vite-env.d.ts:1]: It's a special type of comment that TS reads and understands. _I_ don't understand it personally, but it does and that's all that matters ¯\_(ツ)_/¯

File: examples/get-started/vite.config.ts (hunk 2 above)
Comment (knagurski) [reply examples/get-started/vite.config.ts:4]: This was part of the vite codegen... probably just for reference

File: examples/get-started/src/App.module.css (hunk 6 above)
Comment (knagurski) [reply examples/get-started/src/App.module.css:6]: Welcome to the world of grid templates. This syntax says that we'll have 2 rows and 2 columns and that the "Sections" named grid region will span 2 columns. The `grid-area: Sections` in the `.sections` class tells it that it should plop into this place.

File: examples/get-started/src/components/AllFlagsUsingHOC.tsx (hunk 3 above)
Comment (knagurski) [reply examples/get-started/src/components/AllFlagsUsingHOC.tsx:7]: Nice attention to detail 👍🏻 I was mixed on this as _Feature Flags_, the product, is a "Harness noun", but in this context we're referring to the flags themselves.

Comment (crystal-fong) [reply examples/get-started/src/components/AllFlagsUsingHOC.tsx]: okie doke 👍

**Review by crystal-fong — APPROVED** [review]

**Review by crystal-fong — APPROVED** [review]

**Review by Kira-Jennings — APPROVED** [review]

**Pull request merged**